docs/validator.md

```diff
@@ -84,39 +87,74 @@ pip install torch
 
 5. (Optional) Run a Subtensor instance:
 
-Your node will run better if you are connecting to a local Bittensor chain entrypoint node rather than using Opentensor's. 
-We recommend running a local node as follows and passing the ```--subtensor.network local``` flag to your running miners/validators. 
+Your node will run better if you are connecting to a local Bittensor chain entrypoint node rather than using Opentensor's.
+We recommend running a local node as follows and passing the ```--subtensor.network local``` flag to your running miners/validators.
 To install and run a local subtensor node follow the commands below with Docker and Docker-Compose previously installed.
+
 ```bash
 git clone https://github.com/opentensor/subtensor.git
 cd subtensor
 docker compose up --detach
 ```
----
 
-# Running the Validator
+## Obtaining your Hugging Face token
+
+The dataset for code, `The Stack V1-dedup`, requires a **Hugging Face access token**. Follow these steps to obtain and configure one:
+
+### Step 1: Get Your Hugging Face Access Token
```

Comment:
I think that they first need to go to the dataset page and enter their email address and submit to unblock access

Reply:
added it now!